js/src/cli/actions.ts

```diff
@@ -0,0 +1,80 @@
+import chalk from "chalk";
+import { Command } from "commander";
+import client from "../sdk/client/client";
+import { getOpenAPIClient } from "../sdk/utils/config";
+import { ListActionsV2Data } from "../sdk/client";
+
+export default class ActionCommand {
+  private program: Command;
+
+  constructor(program: Command) {
+    this.program = program;
+    this.program
+      .command("actions")
+      .description("Composio Actions")
+      .option(
+        "-a, --apps <appName>",
+        "List all actions for the given apps",
+        (value, previous: string[]) => previous.concat([value]),
+        []
+      )
+      .option(
+        "--tags <tagName>",
+        "List all actions for the given tags",
+        (value, previous: string[]) => previous.concat([value]),
+        []
+      )
+      .option("--use-case", "Search for actions based on the given use case")
+      .option("--limit <limit>", "Limit the number of actions to display")
+      .option("--enabled", "Only show enabled actions")
+      .option("--copy", "Copy the action to the clipboard")
```

Comment:
The `copy` option is defined in the command options (line 30) but not used in the `handleAction` method. Either implement the copy functionality or remove the unused option.